### PR TITLE
Add quantization

### PR DESCRIPTION
**Pull request by zrphercule**
rt, add foxi quantization.
pls sync this to fbcode asap, thanks.

### CODE REVIEW

File: foxi/onnxifi.h [hunk 1]
```diff
@@ -863,6 +863,22 @@ typedef struct onnxTensorDescriptorV1 {
    * For a scalar, this pointer can be NULL.
    */
   const uint64_t* shape;
+  /**
+   * A flag indicates whether this tensor is a quantized tensor or not.
+   * If it is a quantized tensor, the value should be set to 1.
+   * By default it should be set to 0.
+   */
+  unsigned char is_quantized;
```
Comment (houseroad) [inline foxi/onnxifi.h]: why not bool or uint8_t?

Comment (yinghai) [reply foxi/onnxifi.h]: C?

Comment (zrphercule) [reply foxi/onnxifi.h]: Cause it is C?

Comment (houseroad) [reply foxi/onnxifi.h]: we already use int32_t anyway, why hesitate using uint8_t?

Comment (yinghai) [reply foxi/onnxifi.h]: That's true.

Comment (zrphercule) [reply foxi/onnxifi.h]: Then let me try uint8_t? didnt see the difference...

Comment (houseroad) [reply foxi/onnxifi.h]: it's weird to use char to store the number, which means you need explict conversion. So if it's possible, that's use uint8 instead

Comment (zrphercule) [reply foxi/onnxifi.h]: ok, let me change it.

**Review by houseroad — APPROVED** [review]
LGTM

**Comment by yinghai** [comment]
How to sync it back?

**Pull request merged**